app/src/main/plugins.js

```diff
@@ -125,6 +125,28 @@ class Plugins {
 
     return ret;
   }
+
+  getShareServicesPerFormat() {
+    const ret = {};
+
+    for (const service of this.getShareServices()) {
+      for (const format of service.formats) {
+        ret[format] = ret[format] || [];
+        ret[format].push(service);
+      }
+    }
+
+    return ret;
+  }
+
+  prettifyFormat(format) {
+    const ret = format.toUpperCase();
+    const formats = new Map([
+      ['WEBM', 'WebM']
+    ]);
```

Comment:
I know it's moot, but better for readability to just make a manual mapping of all of them, even if it works fine to just uppercase. Most consistent. Less differing code-paths.